src/types/style.ts

```diff
@@ -4,5 +4,12 @@ export type Leading = 'default' | 'narrow' | 'tight';
 export type TextColor = 'main' | 'sub' | 'link' | 'linkSub' | 'disabled' | 'primary' | 'accent' | 'alert' | 'white';
 export type Spacing = 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl';
 export type FlexDirection = 'row' | 'column';
-export type AlignItems = 'flex-start' | 'center' | 'flex-end' | 'stretch' | 'baseline';
-export type JustifyContent = 'flex-start' | 'center' | 'flex-end' | 'space-between' | 'space-around' | 'space-evenly';
+export type AlignItems = 'normal' | 'flex-start' | 'center' | 'flex-end' | 'stretch' | 'baseline';
```

Comment:
Add `normal`.
Because in the case of `flex-start`, the child elements will be as wide as fit-content